From 52a3ea860b23818b4ca8745078b52e49cebb986e Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Fri, 8 Feb 2019 18:07:51 +0200
Subject: [PATCH 645/706] enetc_mdio: Change naming convention of PCI EP mdio
 API to avoid clashes

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
(cherry picked from commit 9cd349a6fd128b3ba707feca2dcae55a3c16ac5e)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/freescale/enetc_mdio.c | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/drivers/net/ethernet/freescale/enetc_mdio.c b/drivers/net/ethernet/freescale/enetc_mdio.c
index 32c99119f573..490873332bec 100644
--- a/drivers/net/ethernet/freescale/enetc_mdio.c
+++ b/drivers/net/ethernet/freescale/enetc_mdio.c
@@ -39,11 +39,11 @@
 #define ENETC_MDIO_DEV_ID	0xee01
 #define ENETC_DRV_NAME_STR "ENETC MDIO driver"
 
-static int enetc_mdio_probe(struct pci_dev *pdev,
-			    const struct pci_device_id *ent)
+static int enetc_pci_mdio_probe(struct pci_dev *pdev,
+				const struct pci_device_id *ent)
 {
 	const struct xgmac_mdio_cfg cfg = {
-		.bus_name = "Freescale ENETC MDIO Bus",
+		.bus_name = "Freescale PCI ENETC-RCIE MDIO Bus",
 		.regs_offset = 0x1C00,
 	};
 	int err;
@@ -76,26 +76,26 @@ static int enetc_mdio_probe(struct pci_dev *pdev,
 	return err;
 }
 
-void enetc_mdio_remove(struct pci_dev *pdev)
+static void enetc_pci_mdio_remove(struct pci_dev *pdev)
 {
 	xgmac_mdio_remove(dev_get_drvdata(&pdev->dev));
 	pci_release_mem_regions(pdev);
 	pci_disable_device(pdev);
 }
 
-static const struct pci_device_id enetc_mdio_id_table[] = {
+static const struct pci_device_id enetc_pci_mdio_id_table[] = {
 	{ PCI_DEVICE(PCI_VENDOR_ID_FREESCALE, ENETC_MDIO_DEV_ID) },
 	{ 0, } /* End of table. */
 };
 MODULE_DEVICE_TABLE(pci, enetc_mdio_id_table);
 
-static struct pci_driver enetc_mdio_driver = {
+static struct pci_driver enetc_pci_mdio_driver = {
 	.name = KBUILD_MODNAME,
-	.id_table = enetc_mdio_id_table,
-	.probe = enetc_mdio_probe,
-	.remove = enetc_mdio_remove,
+	.id_table = enetc_pci_mdio_id_table,
+	.probe = enetc_pci_mdio_probe,
+	.remove = enetc_pci_mdio_remove,
 };
-module_pci_driver(enetc_mdio_driver);
+module_pci_driver(enetc_pci_mdio_driver);
 
 MODULE_DESCRIPTION(ENETC_DRV_NAME_STR);
 MODULE_LICENSE("Dual BSD/GPL");
-- 
2.17.1

